### PR TITLE
Simplified uVisor handbook page

### PR DESCRIPTION
**Pull request by mbartling**
* Moved big example to *temp* tutorial
* Referenced github example and added small reference
* Described high level usage

@patater What do you think? I want to add some more descriptions from
the high level overview pdf and may be a limitations section. Keep in
mind that the bulk documentation will be on the uVisor site.

### CODE REVIEW

**Comment by Patater** [comment]
Sorry, this doesn't seem like the correct place to make these changes.

The handbook page is copied (hopefully automatically) from the [uVisor quickstart guide ](https://github.com/ARMmbed/uvisor/blob/master/docs/lib/QUICKSTART.md).

Could you make the changes upstream in the uVisor project instead?

Please also make separate git commits for formatting changes and meaningful content changes, as that will make it easier to review and maintain.

Thanks

File: docs/refs/api/security/uvisor.md [hunk 1]
```diff
@@ -1,573 +1,44 @@
...
-Repeat the process multiple times until all ACLs have been added to the list. When no other ACL is needed, the system runs without hitting a uVisor fault.
+###### main-hw.h
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-uvisor)](https://github.com/ARMmbed/mbed-os-example-uvisor/blob/master/source/main-hw.h)
 
-##### Additional resources
+###### led.cpp
 
-- [uVisor API documentation](API.md).
-- [Debugging uVisor on Mbed OS](DEBUGGING.md).
+Describe the compartmentalized application.
```
Comment (sarahmarshy) [inline docs/refs/api/security/uvisor.md]: What does this mean? It also did not display very prettily in "view"

**Comment by sarahmarshy** [comment]
@Patater the handbook entries are being simplified to quick description, API, and example. This conforms to that. This extended documentation is now more suitable for a tutorial.

**Comment by mbartling** [comment]
@Patater I believe this entry should be considered separate from the quickstart. If you want I can add it as an overview page in the uVisor repo.

**Comment by Patater** [comment]
@sarahmarshy @mbartling

OK. Let's keep all the uVisor docs sourced from the uVisor repository, though. It's easier for us to maintain and keep up to date that way.

**Comment by mbartling** [comment]
@Patater I will go ahead and make a PR in uVisor then.

-Michael

**Comment by Patater** [comment]
Thanks!

**Comment by AnotherButler** [comment]
Closing in favor of #511 in uVisor.

**Pull request closed without merging**